src/translator/service.cpp

```diff
@@ -56,45 +56,45 @@ std::vector<Response> Service::translateMultiple(std::vector<std::string> &&inpu
   // We queue the individual Requests so they get compiled at batches to be
   // efficiently translated.
   std::vector<std::future<Response>> responseFutures;
-  for (auto &input : inputs) {
-    std::future<Response> inputResponse = queueRequest(std::move(input), responseOptions);
-    responseFutures.push_back(std::move(inputResponse));
+  for (size_t i = 0; i < inputs.size(); i++) {
+    // https://stackoverflow.com/a/33437008/4565794
+    // @jerinphilip is not proud of this.
+    Ptr<std::promise<Response>> responsePromise = New<std::promise<Response>>();
+    responseFutures.push_back(std::move(responsePromise->get_future()));
```

Comment:
Ugh, let's just not support translateMultiple on native.

Reply:
So going for translateMultiple only on WASM with ifdefs kills the only test which tests mozilla's codepath is intact and works (on native). I will have to skip this to get green -> implying minimal marian has no tests also the same for Mozilla code path. 

WASM workflow only checks for builds on CI, no tests for whether intended translation comes for a given input or not. I'd rather have test coverage for this codepath at the cost of having a promise + future + wait. 

I'm trying to work out a solution on #177. This branch is reset to the last state where all tests pass.